dev-util/kdstatemachineeditor/files/fix_hardcoded_installation_dirs-1.2.1.patch

```diff
@@ -0,0 +1,29 @@
+--- a/CMakeLists.txt
```

Comment:
Is this patch taken from upstream? If so, please add the git formatted version complete with header.

Reply:
I think you are talking about [this](https://github.com/KDAB/KDStateMachineEditor/commit/e6ac7f3ca0ed0fe699d63eba822b94f8486a7e7e.patch).

Reply:
yes